IPG-Automotive-UK/react-ui
# Migrate AlignVertical to typescript

## Changes

- Migrate `AlignVertical` to typescript
- Disabled `onChange` story control as it doesn't make sense to be configurable
- Tidied up `AlignHorizontal` story and docs to be consistent with `AlignVertical`

## Testing notes

* Ensure component acts and behaves as before.
* Ensure docs are correctly parsed from the types
* Ensure types look correct

## Author checklist before assigning a reviewer
